manager-dashboard/app/views/NewTutorial/CustomOptionInput/styles.css

```diff
@@ -0,0 +1,31 @@
+.custom-option {
+    display: flex;
+    flex-direction: column;
+    gap: var(--spacing-large);
+
+    .option-content {
+        display: flex;
+        gap: var(--spacing-medium);
+
+        .option-input {
+            flex-grow: 1;
+        }
+
+        .option-icon {
+            flex-basis: 10rem;
+        }
```

Comment:
Let's take a look into why we use 10rem here? @frozenhelium

Reply:
Because 10rem fits the size properly, it's `option-icon-input`